civis/cli/_cli_commands.py

```diff
@@ -85,6 +86,71 @@ def notebooks_download_cmd(notebook_id, path):
             f.write(lines)
 
 
+@click.command('launch')
+@click.argument('language', type=click.Choice(['python3', 'python2', 'r']),
+                default='python3')
+@click.option('--mem', type=int, default=None,
+              help='Memory allocated for this notebook in MiB.')
+@click.option('--cpu', type=int, default=None,
+              help='CPU available for this notebook in 1/1000 of a core.')
```

Comment:
is it 1024 of a core?  cf. https://platform.civisanalytics.com/#/api#post_scripts_containers

Reply:
Yes, I think it is. Fixed.